schemas/src/registry_schemas/schemas/incorporation.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/incorporation",
+    "required": [
+        "incorporation"
+    ],
+    "type": "object",
+    "title": "Incorporation Filing",
+    "properties": {
+        "incorporation": {
+            "type": "object",
+            "required": [
+                "nameRequest",
```

Comment:
I'm late to the game on this, but can we require nameRequest in the case of numbered companies?

Reply:
never late! Absolutely, thats a great point. As far as the flow is concerned, the only way we will know the incorporating businesses legaltype is via the NameRequest portion of the application. So inside the name request schema ( currently ) legaltype is the only thing required at this point. As far we as know ( aka - a chat with Linda about it ) even numbered companies will need to still indicate a type through this NR flow.  Does it make sense to leave it as required, or possibly have legaltype of requirement of another section perhaps?

Reply:
I suppose it depends whether the numbered co's come in with an NR (you can do an NR for a numbered company in the current systems), or not. If the assumption in our current functionality is that they have an NR, ie: we haven't moved on to numbered co's, then we can leave it as is I suppose and change it if needed when we get to incorporating numbered co's. So I guess I'm fine with it for now. The schema is ever-evolving :)